atom/browser/atom_browser_main_parts.cc

```diff
@@ -156,7 +156,8 @@ void AtomBrowserMainParts::PreMainMessageLoopRun() {
 #if !defined(OS_MACOSX)
   // The corresponding call in macOS is in AtomApplicationDelegate.
   Browser::Get()->WillFinishLaunching();
-  Browser::Get()->DidFinishLaunching();
+  base::DictionaryValue* empty_info = new base::DictionaryValue();
```

Comment:
Will this be leaked since it is never freed and isn't wrapped in a smart pointer?